iroha_client/tests/unstable_network.rs

```diff
@@ -0,0 +1,159 @@
+#[cfg(test)]
+mod tests {
+    use async_std::task;
+    use iroha::{config::Configuration, prelude::*};
+    use iroha_client::{
+        client::{self, Client},
+        config::Configuration as ClientConfiguration,
+    };
+    use iroha_data_model::prelude::*;
+    use rand::seq::SliceRandom;
+    use std::{collections::BTreeSet, thread};
+    use tempfile::TempDir;
+
+    const CONFIGURATION_PATH: &str = "tests/test_config.json";
+    const CLIENT_CONFIGURATION_PATH: &str = "tests/test_client_config.json";
+    const N_PEERS: usize = 7;
+    const MAX_FAULTS: u32 = 2;
+    const N_TRANSACTONS: usize = 20;
+    const OFFLINE_PEERS: usize = 1;
```

Comment:
I would recommend to add another test with 2 offline peers and n peers 7.

Reply:
Okay, I will introduce it in a separate PR after we implement this [RFC](https://wiki.hyperledger.org/display/iroha/Change+View+Drawback+in+Sumeragi)